test/integration/targets/run_modules/runme.sh

```diff
@@ -4,3 +4,11 @@ set -eux
 
 # test running module directly
 python.py library/test.py args.json
+
+TMPFILE=$(shell mktemp -p "${OUTPUT_DIR}" 2>/dev/null || mktemp -t 'ansible-testing-XXXXXXXXXX' -p "${OUTPUT_DIR}")
+
+# ensure 'command' can use 'raw args'
+ansible -m command -a "dd if=/dev/zero of=\"${TMPFILE}\" bs=1K count=1" localhost
+
+# ensure fqcn 'command' can use 'raw args'
+ansible -m ansible.legacy.command -a "dd if=/dev/zero of=\"${TMPFILE}\" bs=1K count=1" localhost
```

Comment:
I can only reproduce the bug with a task in a file. The adhoc task ds ends up injecting args as `_raw_params` which avoids the issue.